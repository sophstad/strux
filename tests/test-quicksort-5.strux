int main() {
    int[11] a = [1, 4, 3, 6, 7, 2, 99, 23, 37, 9, 41];
  a.quickSort();
    int i = 0;
    for (i = 0; i < 11; i = i + 1) {
        print(a[i]);
    }
    return 0;
}
