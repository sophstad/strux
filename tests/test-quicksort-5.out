[1, 2, 3, 4, 6, 7, 9, 23, 37, 41, 99]